docs/modules/domain_record.md

```diff
@@ -22,9 +22,19 @@ NOTE: Domain records are identified by their name, target, and type.
 
 ```yaml
 - name: Delete a domain record
-  linode.cloud.domain:
+  linode.cloud.domain_record:
```

Comment:
Here I changed into a right sample, however, the doc validation wants to change it back to the wrong one: https://github.com/linode/ansible_linode/actions/runs/7923038658/job/21631933475?pr=469 Trying to figure it out.